topics/geospatial.md

```diff
@@ -5,12 +5,12 @@ description: >
 ---
 
 Geospatial indexes let you store coordinates and search for them.
-This data structure is useful for finding nearby points within a given radius or bounding box.
+This data structure is useful for finding nearby points within a given radius, bounding box, or "polygon" shaped area.
 
 ## Basic commands
 
 * `GEOADD` adds a location to a given geospatial index (note that longitude comes before latitude with this command).
-* `GEOSEARCH` returns locations with a given radius or a bounding box.
+* `GEOSEARCH` returns locations with a given radius, bounding box, or "polygon" shaped area enclosed by polygon vertices.
```

Comment:
Skip the quotes? (Or is it not a real polygon because it's on a curved surface? In that case, we could use write "box" in quotes too since it's not a real box either...)

How to specify the polygon is part of the syntax details so we can skip this in the high-level description.

```suggestion
* `GEOSEARCH` returns locations with a given radius, bounding box, or polygon shaped area.
```